ykman-gui/py/yubikey.py

```diff
@@ -210,6 +210,13 @@ def refresh_piv(self):
             session = PivSession(conn)
             pivman = get_pivman_data(session)
 
+            if self._dev_info['version'].split('.') >= ['5', '3', '0']:
```

Comment:
Instead of doing an explicit version check here, call:
```
try:
    key_type = session.get_management_key_metadata().key_type
except NotSupportedError:
    key_type = ...
key_length = key_type.key_len
```
That way the version check isn't duplicated, since it's already in `get_management_key_metadata`.
Also don't call get_management_key_metadata twice when it isn't needed.